listener.go

```diff
@@ -57,6 +72,7 @@ func WrapListenerIncludingErrorHandler(wrapped net.Listener, pool BufferPool, se
 		onError:          onError,
 		connCh:           make(chan net.Conn),
 		errCh:            make(chan error),
+		initMsgTimeout:   defaultInitMsgTimeout,
```

Comment:
We currently will always use the default value.  I did this mainly because I did not want to make a breaking change to the public API (by adding a function parameter).  I don't know that we'd have a strong need to set this value to anything specific anyway.